control/modelsimp.py

```diff
@@ -50,7 +50,7 @@
 from .statesp import StateSpace
 from .statefbk import gram
 
-__all__ = ['hsvd', 'balred', 'modred', 'era', 'markov', 'minreal']
+__all__ = ['hsvd', 'balred', 'modred', 'era', 'markov', 'minreal',]
```

Comment:
Why did you add a trailing `,` in this list expression? It is not a syntax error, but it is not consistent with the current style of the `control` package.